ui/src/app/views/resource-card/component.css

```diff
@@ -82,7 +82,7 @@
 
 .card .footer {
   position: absolute;
-  bottom: .01rem;
+  bottom: -0.05rem;
```

Comment:
I was so confused as to where the card border comes from. After digging a bit it runs out it's actually being applied via Clarity's CSS, where it puts a `1px` border-bottom on to `.card`s. So I'd use -1px here instead to match that, and put a comment as well - please.

https://github.com/vmware/clarity/blob/56d3f500ed6db1e5a3ffc3442b65ad4a8337997f/src/clr-angular/layout/_card.clarity.scss#L45

Reply:
Done! PTAL